src/collections/BuiltInPages/index.ts

```diff
@@ -0,0 +1,45 @@
+import type { CollectionConfig } from 'payload'
+
+import { byGlobalRole } from '@/access/byGlobalRole'
+import { accessByTenantRole, byTenantRole } from '@/access/byTenantRole'
+import { filterByTenant } from '@/access/filterByTenant'
+import { contentHashField } from '@/fields/contentHashField'
+import { tenantField } from '@/fields/tenantField'
+import { populatePublishedAt } from '@/hooks/populatePublishedAt'
+
+export const BuiltInPages: CollectionConfig<'pages'> = {
+  slug: 'builtInPages',
+  labels: {
+    singular: 'Built-In Page',
+    plural: 'Built-In Pages',
+  },
+  access: {
+    create: accessByTenantRole('builtInPages')?.create,
+    read: byTenantRole('read', 'builtInPages'),
+    update: byGlobalRole('update', 'builtInPages'),
```

Comment:
I'd almost argue that this should be even more locked down, since editing this in the UI is never the right thing to be doing - if we change the set of pages (or the tenant adds a new zone, etc), then something should re-generate this collection automatically, since this data is truly read-only.

Reply:
(not sure it needs to be literally read-only or something, but having edit powers as a user account seems wrong)